js/liveaw.js

```diff
@@ -76,11 +76,12 @@ function list_matches(s, cb) {
 }
 
 function on_edit_event(s) {
-	var match_players = {};
-	s.event.matches.forEach(function(match) {
+	var match_players = s.event.matches.reduce(function(prev, match) {
```

Comment:
Huh? Isn't `forEach` more appropriate here? The initial object is returned 100% of the time.